suite/tests/CMakeLists.txt

```diff
@@ -4272,7 +4272,7 @@ if (BUILD_CLIENTS)
     if (DR_HOST_X86 AND DR_HOST_X64 AND LINUX)
       torunonly_drcacheoff(opcode_categories allasm_x86_64 ""
         "@-simulator_type@opcode_mix" "")
-
+    
```

Comment:
@derekbruening There are some trailing spaces that cannot be removed,  I think the trailing space checker reenabled recently should ignore diff lines starting with a `-`.

Reply:
That makes sense: file an issue or send a PR?  (I remember the trailing space checker also complaining when checking in actual .diff files (e.g. patches for elfutils) and having to disable it during PR testing.)

Reply:
Okay, I will send a PR.